src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -216,14 +216,14 @@ internal Font(IntPtr nativeFont, string familyName, FontStyle style, float size)
                 fontFamily = FontFamily.GenericSansSerif;
             }
 
-            setProperties(fontFamily, size, style, GraphicsUnit.Pixel, 0, false);
+            Initialize(fontFamily, size, style, GraphicsUnit.Pixel, 0, false);
```

Comment:
I don't know much about this codebase but shouldn't the last parameter be `IsVerticalName(fontFamily.Name)` instead of false? cc @safern